File: Python/FC4.py
import math

def convert(n):
    if n > 0:
        numbers=[]
        for d in range(1, math.floor(math.sqrt(n)+1)):
            if (n % d == 0):
                numbers.extend([int(n/d),d])
        numbers.sort()
        return(numbers)
    else:
        return("Error!")

#convert(int(input("What is the number?\n")))

File: Java/FC4.java
import java.util.Scanner;
class FC4 {
    public static void main(String[] args) {
        System.out.println("What is the number?");
        Scanner in = new Scanner(System.in);
        long n = in.nextLong();
        if (n > 0)
        {
            for (long d=1; d<Math.sqrt(n); d++)
                if ((n % d) == 0)
                {
                    System.out.println(n / d + ", " + d);
                }
        }
        else
        {
        System.out.println("Please try again.");
        }
    }
}